### DISCUSSION TITLE
Penguinmod Package issue?

### DISCUSSION THREAD

**Discussion by Never-Developer** [cloudlink-omega/.github#4] [General]
![image](https://github.com/user-attachments/assets/ee7f1b50-bf90-427b-baf7-169c39b598de)
![image](https://github.com/user-attachments/assets/86390832-2968-471b-9677-94bd8cbf52e5)
![image](https://github.com/user-attachments/assets/e7ffe8d4-9eef-4b10-ad24-8894ffaa22c4)
![image](https://github.com/user-attachments/assets/d8a6d09e-f9c0-436c-900d-3a8373d75290)

**Comment by MikeDev101** [comment]
This is likely a CORS related issue. Can you provide me a full screenshot of the console log?

**Reply by Never-Developer** [reply]
https://drive.google.com/file/d/1oM5_2mq5S3uqFcD1A_XOUPzyBGGTC1K0/view?usp=sharing 
I shared video sir
